GUI/src/cz/fidentis/gui/StartingPanel.java

```diff
@@ -304,28 +298,7 @@ public static void createNewProject() {
             Project project = new Project("Project " + dateFormat.format(date));
             project.setName("Project " + dateFormat.format(date));
             try {
-                if(!triedDeleting){
-                    triedDeleting = true;
-                    
-                    ObservableMaster master = new ObservableMaster();
-                    final ProgressHandleObserver obs = new ProgressHandleObserver("Deleting tmp files");
-                    master.addObserver(obs);
-                
-                    Runnable run = new Runnable() {
-                        @Override
-                        public void run() {
-                            obs.startHandle();
-                        }
-                    };
-                
-                    Thread t = new Thread(run);
-                    t.start();                    
-                
-                    project.setTempDirectory(FileUtils.instance()
-                        .createTMPmoduleFolder(String.valueOf(System.currentTimeMillis())));
-                
-                    master.updateObservers();
-                }
+                project.setTempDirectory(FileUtils.instance().createTMPmoduleFolder(String.valueOf(System.currentTimeMillis())));
```

Comment:
len pripomienka, prvy krat ked sa zavola FileUtils.instance() vymaze sa predchadzajuci tmp folder, ak tam ostal z predchadzajucich behov fidentisu a vytvori sa nova zlozka pre tento beh programu. Pokial chces mazanie spravit explicitne v triede Install (tusim), tak to tam nezabudni hodit predtym, ako sa mergne PR (za okolnosti, ked bude tych suborov na mazanie vela, tak teraz program zamrzne a nebude existovat ziaden sposob oznamenia uzivatelovi, ze program pracuje a nezamrzol)